web/packages/operations/ecosystem.config.js

```diff
@@ -3,7 +3,20 @@ module.exports = {
     {
       name: "monitor",
       node_args: "--require=dotenv/config",
-      script: "./dist/src/cron.js",
+      script: "./dist/src/main.js",
+      args: "cron"
+    },
+    {
+      name: "transferToPolkadot",
+      node_args: "--require=dotenv/config",
+      script: "./dist/src/transfer_to_polkadot.js",
+      args: "cron"
+    },
+    {
+      name: "transferToEthereum",
+      node_args: "--require=dotenv/config",
+      script: "./dist/src/transfer_to_ethereum.js",
+      args: "cron"
```

Comment:
Periodically run the transfer tests.

Reply:
How do we know if they fail?

Reply:
Yeah, I remove the wait logic there. Instead we depend on alarming on relayer logs & bridge/channel status as we do for the production.

Alarms forwarded to https://snowfork.slack.com/archives/C07BG558XE1